rs/pocket_ic_server/src/main.rs

```diff
@@ -78,6 +78,7 @@ fn current_binary_path() -> Option<PathBuf> {
     std::env::args().next().map(PathBuf::from)
 }
 
+/// Some bogus comment to trigger a build
```

Comment:
Are you going to remove the comment again?

Reply:
Yes, once it works. I turned the PR back to draft since it needs more work.